src/plugins/Workers/SyncWorker/utils/TransactionOrderer/methods/insertAtPos.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Add a transaction at a specific position
+ * @param {Transaction} transaction
+ * @param {number?} pos
+ * @return this
+ */
+module.exports = function insertAtPos(transaction, pos) {
```

Comment:
`insertAtPos(transaction, pos = null)` ?

Reply:
Can you recheck the function definition of the last commit ?